src/coreclr/debug/di/rspriv.h

```diff
@@ -2879,6 +2879,10 @@ class IProcessShimHooks
     virtual void RequestSyncAtEvent()= 0;
 
     virtual bool IsThreadSuspendedOrHijacked(ICorDebugThread * pThread) = 0;
+
+#ifdef FEATURE_INTEROP_DEBUGGING
+    virtual bool IsUnmanagedThreadHijacked(ICorDebugThread * pICorDebugThread) = 0;
```

Comment:
Why did you need this here and something very similar on line 3470?

Reply:
> Why did you need this here and something very similar on line 3470?

We need to access the `IsUnmanagedThreadHijacked` API from `ShimStackWalk`, which stores `CordbProcess` via a `IProcessShimHooks` interface:
 https://github.com/dotnet/runtime/blob/432c4a09418b52cc8b19c385504f14aae2a39038/src/coreclr/debug/di/shimpriv.h#L571

Since we do not wish to add a new public API to `ICorDebugProcess`, the `IsUnmanagedThreadHijacked` API must be declared in both the `IProcessShimHooks` interface  as well as the `CordbProcess` implementation class in rspriv.h for `ShimStackWalk` to use it.  

Hope this helps!